core/src/kura.rs

```diff
@@ -115,7 +115,7 @@ impl Kura {
     /// - file storage is unavailable
     /// - data in file storage is invalid or corrupted
     #[iroha_logger::log(skip_all, name = "kura_init")]
-    pub fn init(&self) -> Result<Vec<HashOf<VersionedCommittedBlock>>> {
+    pub fn init(&self) -> Result<usize> {
```

Comment:
Can you please add a doc comment to explain what does this return type mean?

Reply:
Or express it as a newtype, like `BlockCount(usize)`, so that it will be literally a self-documenting code.